Entities/MOSRotating.cpp

```diff
@@ -1402,8 +1403,10 @@ bool MOSRotating::DeepCheck(bool makeMOPs, int skipMOP, int maxMOPs)
 void MOSRotating::PreTravel() {
 	MOSprite::PreTravel();
 
+#ifdef DRAW_MOID_LAYER
```

Comment:
A fun advantage of this change is it actually improves accuracy with fast-moving MOIDs, as we don't need this optimization to not redraw when moving too fast. In fact, redrawing is never required, as MOIDs can now overlap.